src/chart/graph/GraphSeries.ts

```diff
@@ -69,7 +69,7 @@ export interface GraphNodeStateOption<TCbParams = never> {
 
 
 interface ExtraEmphasisState {
-    focus?: DefaultEmphasisFocus | 'adjacency'
+    focus?: DefaultEmphasisFocus | 'adjacency' | 'trajectory'
```

Comment:
Why do you add `'trajectory'` for the `graph` series? I think you are just adding support for the `sankey` series. If you also plan to support this feature for the `graph` series, it's not enough to only add typings.